source/content/guides/trello.md

```diff
@@ -17,9 +17,7 @@ Be sure that you:
 
 - Have a Drupal or WordPress site on Pantheon
 - [Install Terminus](/terminus/install)
-- [Generate a Machine Token](https://dashboard.pantheon.io/machine-token/create) from **User Dashboard** > **Account** > **Machine Tokens**, then authenticate Terminus:
-
-        terminus auth:login --machine-token=‹machine-token›
```

Comment:
Why remove the `terminus auth:login` here?

Reply:
It was a merge commit. I accepted that change from the upstream branch